client/src/helpers/authFetch.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Create a wrapper function that add the token as config into fetch() function
+ */
+
+export const authFetch = (...args) => {
+    let token = localStorage.getItem('access_token') || null
+    let config = {}
+
+    if(token) {
+        config = {
+            headers: { 'Authorization': `Bearer ${token}` }
+        }
+    }
+    else {
+        throw "No token saved! Please login again.";
+        //redirect to login page
+        // window.location.replace("/login")
+        console.log("no token saved")
+        return;
+    }
+
+    return fetch(...args, config);
```

Comment:
Given that the token may or may not be valid, I think you should handle that possibility here. You could even make this function more robust by checking the response status and throwing an error if it's >400, parsing the JSON response, etc.